client/src/pages/tasks/Show.js

```diff
@@ -293,6 +293,29 @@ const TaskShow = ({ pageDispatchers }) => {
                       }
                     />
                   )}
+                  {task.descendantTasks && task.descendantTasks.length > 0 && (
+                    <TaskParentTask
+                      dictProps={Settings.fields.task.childTask}
+                      name="childTask"
+                      component={FieldHelper.ReadonlyField}
+                      humanValue={
+                        task.descendantTasks && task.descendantTasks.length > 0 && (
```

Comment:
The check for `.length` is not needed here (and if it was, you could use optional chaining). In fact, you could simply do: `humanValue={task?.descendantTasks?.map(…)}`